datasets/dataforgood-fb-hrsl.yaml

```diff
@@ -0,0 +1,27 @@
+Name: High Resolution Population Density Maps + Demographic Estimates by CIESIN and Facebook
+Description: |
+  The population of the world, allocated to 1 arcsecond blocks. This
+  refines CIESIN’s Gridded Population of the World project, using
+  machine learning models on high-resolution worldwide Digital Globe
+  satellite imagery.
+Documentation: https://dataforgood.fb.com/tools/population-density-maps/
+Contact: disastermaps@fb.com
+UpdateFrequency: Quarterly
+Tags:
```

Comment:
Based on our current tags (we don't tag by country or company name), I'd recommend the below list.

 - population
  - demographics
  - machine learning
  - aerial imagery
  - satellite imagery
  - image processing
  - geospatial
  - sustainability
  - disaster response
  - aws-pds

I'll add population and demographics to the list of allowed tags now.